packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -159,7 +179,23 @@ function listenClickEvents(callback: (clickEvent: MouseEvent & { target: Element
   )
 }
 
-function newPotentialClickAction(
+const enum ClickStatus {
+  // Initial state, the click is still ongoing.
+  PENDING,
+  // The click is no more ongoing but still needs to be validated or discarded.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** unify naming‏
```suggestion
  // Initial state, the click is still ongoing.
  ONGOING,
  // The click is no more ongoing but still needs to be validated or discarded.
```